### PR TITLE
In AboutScreen, remove unnecessary padding and add parallax

### PR DESCRIPTION
**Pull request by kosenda**
## Issue
no issue

## Overview (Required)
- I looked at the [Figma](https://www.figma.com/design/XUk8WMbKCeIdWD5cz9P9JC/DroidKaigi-2024-App-UI?node-id=54982-75342&t=l2TqUXx0gVgB367Z-4) and removed padding that I thought was unnecessary.
- (Suggestion) I thought it would be a richer UI if we added a parallax to AboutHeader

## Movie
Before | After
:--: | :--:
<video src="https://github.com/user-attachments/assets/d7416533-cd8e-4c0e-bb5d-db01c92e16b2" width="300" > | <video src="https://github.com/user-attachments/assets/b623c19a-627b-4962-b156-85fd15d54455" width="300" >

## Detail
I chose 40.dp because the space between the header image and text was 40 in Figma

https://github.com/user-attachments/assets/4babc926-8f8d-47b1-8a95-48471f0703bd

https://www.figma.com/design/XUk8WMbKCeIdWD5cz9P9JC/DroidKaigi-2024-App-UI?node-id=54982-75342&t=l2TqUXx0gVgB367Z-4

### CODE REVIEW

File: feature/about/src/commonMain/kotlin/io/github/droidkaigi/confsched/about/section/AboutDroidKaigiDetail.kt [hunk 1]
```diff
@@ -22,6 +28,9 @@ import io.github.droidkaigi.confsched.designsystem.theme.KaigiTheme
 import org.jetbrains.compose.resources.painterResource
 import org.jetbrains.compose.resources.stringResource
 import org.jetbrains.compose.ui.tooling.preview.Preview
+import kotlin.math.roundToInt
+
+const val MAX_ABOUT_HEADER_OFFSET_DP = 40
```
Comment (takahirom) [inline feature/about/src/commonMain/kotlin/io/github/droidkaigi/confsched/about/section/AboutDroidKaigiDetail.kt:33]: Thanks. We are using upper camel case like this. And can we make this private?
https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#singletons-constants-sealed-class-and-enum-class-values

File: feature/about/src/commonMain/kotlin/io/github/droidkaigi/confsched/about/section/AboutDroidKaigiDetail.kt [hunk 2]
```diff
@@ -30,9 +39,23 @@ object AboutDetailSectionTestTag {
 
 @Composable
 fun AboutDroidKaigiDetail(
+    state: LazyListState,
```
Comment (takahirom) [inline feature/about/src/commonMain/kotlin/io/github/droidkaigi/confsched/about/section/AboutDroidKaigiDetail.kt:42]: I don't think this is not a main state for AboutDroidKaigiDetail. So I think we have to consider the name 👀 like screenScrollState or something like that

File: feature/about/src/commonMain/kotlin/io/github/droidkaigi/confsched/about/section/AboutDroidKaigiDetail.kt (hunk 1 above)
Comment (kosenda) [reply feature/about/src/commonMain/kotlin/io/github/droidkaigi/confsched/about/section/AboutDroidKaigiDetail.kt]: I apologies 🙇  I’ve fix this.

File: feature/about/src/commonMain/kotlin/io/github/droidkaigi/confsched/about/section/AboutDroidKaigiDetail.kt (hunk 2 above)
Comment (kosenda) [reply feature/about/src/commonMain/kotlin/io/github/droidkaigi/confsched/about/section/AboutDroidKaigiDetail.kt]: I thought you were right, so I changed it to `screenScrollState` !

**Review by takahirom — APPROVED** [review]
You nailed it! Thanks you for your contribution!

**Pull request merged**